### DISCUSSION TITLE
Need to disable/hide edit and delete button on dashboard listing screen

### DISCUSSION THREAD

**Discussion by michaelkovatt** [apache/superset#26109] [Q&A / Help]
I have a requirement where I need to disable/hide edit and delete button. Is there a way to achieve this using superset permissions or any other workaround ?

The links are highlighted below in yellow.
<img width="883" alt="Untitled" src="https://github.com/apache/superset/assets/151566993/5f265c0d-f793-4691-9774-4b32d60ff377">
Context below:
- User is assigned alpha role and is owner of the dashboard.
- Superset version: 2.1.0
- Access to dashboard managed through DASHBOARD_RBAC [feature flag](https://superset.apache.org/docs/installation/configuring-superset#feature-flags)

**Comment by mani-jk** [comment]
Did u found the solution?
